neo/Prompt/Commands/Show.py

```diff
@@ -238,8 +239,11 @@ def execute(self, arguments=None):
             bpm = diff / mins
             tps = Blockchain.Default().TXProcessed / secs
 
+        syncmngr = SyncManager()
+        l_block_cache = syncmngr.block_cache_count
+
         out = "Progress: %s / %s\n" % (height, headers)
-        out += "Block-cache length %s\n" % Blockchain.Default().BlockCacheCount
+        out += "Block-cache length %s\n" % l_block_cache
```

Comment:
i think a property is a bit of overkill for just this function. We can get the info directly
```suggestion
        out += "Block-cache length %s\n" % len(syncmngr.block_cache)
```